biomage/utils/constants.py

```diff
@@ -22,3 +22,9 @@
 SAMPLES_TABLE = "samples"
 EXPERIMENTS_TABLE = "experiments"
 PROJECTS_TABLE = "projects"
+
+# S3 Bucket names
+SAMPLES_BUCKET = "biomage-originals"
+RAW_RDS_BUCKET = "biomage-source"
+PROCESSED_RDS_BUCKET = "processed-matrix"
+CELLSETS_BUCKET = "cell-sets"
```

Comment:
This RDS can be confusing. Can you rename them to `RAW_FILES_BUCKET` and `PROCESSED_FILES_BUCKET`? Unless you have another naming idea?

Reply:
I'll rename it to `RAW_SEURAT_BUCKET` and `PROCESSED_SEURAT_BUCKET`

Reply:
I'll go with your suggestion Iva